transform/models/marts/engaged_ca/ec__events_form_submit.sql

```diff
@@ -22,7 +23,7 @@ form_submits as (
             else split_part(page_location, '?', 1)
         end as page_location_of_event,
         referral_source,
-        total_event_count
+        sum(total_event_count) as total_event_count
```

Comment:
I'd say this is one of the dangers of using `group by all`: anything that doesn't have an aggregation implicitly becomes part of the group by keys. Maybe a good reason to avoid using it (despite how nice it can be)